firmware/Core/Src/telecommands/telecommand_executor.c

```diff
@@ -117,9 +117,10 @@ uint8_t TCMD_execute_parsed_telecommand_now(const uint16_t tcmd_idx, const char
     }
     TCMD_TelecommandDefinition_t tcmd_def = TCMD_telecommand_definitions[tcmd_idx];
 
-    DEBUG_uart_print_str("======== Executing telecommand '");
+    DEBUG_uart_print_str("=========================");
+    DEBUG_uart_print_str(" Executing telecommand '");
     DEBUG_uart_print_str(tcmd_def.tcmd_name);
-    DEBUG_uart_print_str("' ========\n");
```

Comment:
Single quote is dropped here

Reply:
Ohh yeah. Ill add it. Totally missed that one. Ill make use of diff on github online henceforth. It will be a good resource to note where I made changes for sure.

When you say the line lengths are not the same, do you mean the number of characters or the string length?

Reply:
I did a manual count and made sure the `=` are 25 in each `DEBUG_uart_print_str` call i make for the delimiter